packages/dx-react-scheduler/src/plugins/all-day-panel.tsx

```diff
@@ -70,7 +73,7 @@ class AllDayPanelBase extends React.PureComponent<AllDayPanelProps, AllDayPanelS
       messages,
     } = this.props;
     const { rects, elementsMeta } = this.state;
-    const getMessage = getMessagesFormatter({ ...defaultMessages, ...messages });
+    const getMessage = this.getMemoMessageFormatter(messages);
```

Comment:
We should use only pure functions.

`this.getMemoMessageFormatter(messages, defaultMessages)`